fix: force trivy db repository

phiz71 opened the pull request:
By using this repository, we hope reducing the rate-limit error in our CircleCI jobs

phiz71 committed:
refactor: extract fs-scan in a command

graviteeio commented:
Your development orb(s) have been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link(s): 
https://circleci.com/developer/orbs/orb/gravitee-io/aquasec?version=dev:07104876a484bbe6fa1d3a697aa123a021a4f4e6
https://circleci.com/developer/orbs/orb/gravitee-io/aquasec?version=dev:alpha

phiz71 committed:
fix: force trivy db repository

By using this repository, we hope reducing the rate-limit error in our CircleCI jobs

graviteeio commented:
Your development orb(s) have been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link(s): 
https://circleci.com/developer/orbs/orb/gravitee-io/aquasec?version=dev:f3b7feec760e83882a99c798f5cc11c5cb74de0f
https://circleci.com/developer/orbs/orb/gravitee-io/aquasec?version=dev:alpha

phiz71 committed:
fix: force trivy java db repository

By using this repository, we hope reducing the rate-limit error in our CircleCI jobs

graviteeio commented:
Your development orb(s) have been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link(s): 
https://circleci.com/developer/orbs/orb/gravitee-io/aquasec?version=dev:e47d1c77ce4cb214f9825613364caed5265db303
https://circleci.com/developer/orbs/orb/gravitee-io/aquasec?version=dev:alpha
